manifests/monolithic.pp

```diff
@@ -0,0 +1,18 @@
+# **NOTE: THIS IS A [PRIVATE](https://github.com/puppetlabs/puppetlabs-stdlib#assert_private) CLASS**
+#
+# This is simply present to isolate the logic of the installation from the
+# internals
+class stunnel::monolithic {
+  assert_private()
+
+  include '::stunnel'
+
+  contain '::stunnel::config'
```

Comment:
Seems odd that stunnel::config includes stunnel::monolithic and stunnel::monolithic contains stunnel::config.

Reply:
It is! But, you can technically grab either one independently and they both need each other so I'm not really sure what to do about that one.

This will go away when we drop legacy mode and the `monolithic` class is private anyway so it shouldn't cause any harm.

Reply:
Digging a little deeper, you can get to `monolithic` either through `stunnel::config` **or** `stunnel::connection` so you need the reference in both places.